description: fix undefined strerror_r compile error
author: Andres Salomon <dilinger@debian.org>

NOTE: this fix is now upstream in the quiche repo:
https://quiche.googlesource.com/quiche/+/a72e0f686c33b4ba8e134a61a154ae0ffefa74aa%5E%21/epoll_server/simple_epoll_server.cc

This patch can be dropped once chromium syncs their quiche repo and
releases a new version.

--- a/net/third_party/quiche/src/epoll_server/simple_epoll_server.cc
+++ b/net/third_party/quiche/src/epoll_server/simple_epoll_server.cc
@@ -4,6 +4,7 @@
 
 #include "epoll_server/simple_epoll_server.h"
 
+#include <string.h>
 #include <errno.h>   // for errno and strerror_r
 #include <stdlib.h>  // for abort
 #include <unistd.h>  // For read, pipe, close and write.
